QModManager/Patching/Patcher.cs

```diff
@@ -113,6 +111,20 @@ internal static void Patch()
                     return;
                 }
 
+                /*
+                if (PirateCheck.PirateDetected)
+                {
+                    Dialogs.Add(new Dialog()
+                    {
+                        message = "Ahoy,matey! Ye be a pirate!",
+                        leftButton = Dialog.Button.Pirate,
+                        rightButton = Dialog.Button.Pirate,
+                        color = Dialog.DialogColor.Red,
+                    });
+
+                    return;
+                }*/
+
```

Comment:
Don't commit commented code, just remove it :)
```suggestion
```

Reply:
This is gonna be uncommented at some point, it's new code.

Reply:
put space after the comma, before "matey" it should be "Ahoy, matey" not "Ahoy,matey"

Reply:
> This is gonna be uncommented at some point, it's new code.

Then it should be in a branch!